src/abstractions/README.md

```diff
@@ -0,0 +1,41 @@
+# Kiota Abstractions Library for dotnet
+
+[![Build, Test, CodeQl](https://github.com/microsoft/kiota-abstractions-dotnet/actions/workflows/build-and-test.yml/badge.svg?branch=main)](https://github.com/microsoft/kiota-abstractions-dotnet/actions/workflows/build-and-test.yml) [![NuGet Version](https://buildstats.info/nuget/Microsoft.Kiota.Abstractions?includePreReleases=true)](https://www.nuget.org/packages/Microsoft.Kiota.Abstractions/)
```

Comment:
the badge will probably need to be updated, or moved to the main page.

Reply:
Will remove them all and have a section for all the packages published in the main ReadMe file.